src/_includes/partials/footer.njk

```diff
@@ -1,40 +1,56 @@
 
 <footer id="wb-info">
-	<div class="landscape">
-		<nav class="container wb-navcurr">
-			<h2 class="wb-inv">{{ footer[locale].aboutGovernment }}</h2>
-			<ul class="list-unstyled colcount-sm-2 colcount-md-3">
-				<li><a href="{{ footer[locale].contactUsURL }}">{{ footer[locale].contactUs }}</a></li>
-				<li><a href="{{ footer[locale].departmentsAndAgenciesURL }}">{{ footer[locale].departmentsAndAgencies }}</a></li>
-				<li><a href="{{ footer[locale].publicServiceAndMilitaryURL }}">{{ footer[locale].publicServiceAndMilitary }}</a></li>
-				<li><a href="{{ footer[locale].newsURL }}">{{ footer[locale].news }}</a></li>
-				<li><a href="{{ footer[locale].treatiesLawsAndRegulationsURL }}">{{ footer[locale].treatiesLawsAndRegulations }}</a></li>
-				<li><a href="{{ footer[locale].governmentWideReportingURL }}">{{ footer[locale].governmentWideReporting }}</a></li>
-				<li><a href="{{ footer[locale].primeMinisterURL }}">{{ footer[locale].primeMinister }}</a></li>
-				<li><a href="{{ footer[locale].howGovernmentWorksURL }}">{{ footer[locale].howGovernmentWorks }}</a></li>
-				<li><a href="{{ footer[locale].openGovernmentURL }}">{{ footer[locale].openGovernment }}</a></li>
-			</ul>
-		</nav>
+	<h2 class="wb-inv">{{ footer[locale].aboutThisSite }}</h2>
```

Comment:
The `aboutThisSite` value was removed in [src/_data/footer.js](https://github.com/gc-da11yn/gc-da11yn.github.io/pull/324/files#diff-b4613a2f8000afa0de5634f1e6a1d79a3f7590d5a474c04fdb6e896284db05da), leaving an empty `h2`.

Reply:
Good catch, thanks! I added the changes in the latest commit